.github/workflows/development-environment.yml

```diff
@@ -87,7 +87,7 @@ jobs:
 
       - name: Install pyenv
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv
+          HOMEBREW_NO_AUTO_UPDATE=1 HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1 brew install pyenv
```

Comment:
this flag skips upgrading packages that are depended on by a formula -- usually this is fine (we don't care if python@3.12 is actually 3.12.0 instead of 3.12.1)